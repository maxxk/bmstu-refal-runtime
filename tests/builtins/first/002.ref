$ENTRY Go
{
	= <Prout <First 8 'Offset' (1 ) (2) (3) (4) (5) rest expr 'asd' >>
}