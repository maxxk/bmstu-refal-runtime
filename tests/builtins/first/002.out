(Offset(1 )(2 ))(3 )(4 )(5 )rest expr asd
